chainer/links/connection/convolution_nd.py

```diff
@@ -0,0 +1,90 @@
+import math
+
+from chainer.functions.connection import convolution_nd
+from chainer import initializers
+from chainer import link
+
+
+class ConvolutionND(link.Link):
+    """N-dimensional convolution layer.
+
+    This link wraps the :func:`~chainer.functions.convolution_nd` function and
+    holds the filter weight and bias vector as parameters.
+
+    Args:
+        ndim (int): Number of spatial dimensions.
+        in_channels (int): Number of channels of input arrays.
+        out_channels (int): Number of channels of output arrays.
+        ksize (int or tuple of ints): Size of filters (a.k.a. kernels).
+            ``ksize=k`` and ``ksize=(k, k, ..., k)`` are equivalent.
+        stride (int or tuple of ints): Stride of filter application.
+            ``stride=s`` and ``stride=(s, s, ..., s)`` are equivalent.
+        pad (int or tuple of ints): Spatial padding width for input arrays.
+            ``pad=p`` and ``pad=(p, p, ..., p)`` are equivalent.
+        wscale (float): Scaling factor of the initial weight.
+        bias (float): Initial bias value.
+        nobias (bool): If ``True``, then this link does not use the bias term.
+        use_cudnn (bool): If ``True``, then this link uses cuDNN if available.
+        initialW (N-D array): Initial weight value. If ``None``, then this
+            function uses to initialize ``wscale``.
+            May also be a callable that takes ``numpy.ndarray`` or
+            ``cupy.ndarray`` and edits its value.
+        initial_bias (1-D array): Initial bias value. If ``None``, then this
+            function uses to initialize ``bias``.
+            May also be a callable that takes ``numpy.ndarray`` or
+            ``cupy.ndarray`` and edits its value.
+
+    .. seealso::
+        See :func:`chainer.functions.convolution_nd` for the definition of
+        N-dimensional convolution.
+        See :func:`chainer.functions.convolution_2d` for the definition of
+        two-dimensional convolution.
+
+    Attributes:
+        W (~chainer.Variable): Weight parameter.
+        b (~chainer.Variable): Bias parameter.
+
+    """
+
+    def __init__(self, ndim, in_channels, out_channels, ksize, stride=1, pad=0,
+                 wscale=1, bias=0, nobias=False, use_cudnn=True,
+                 initialW=None, initial_bias=None):
```

Comment:
I think the API of `__init__` can be simpler. I guess this argument list is made so that it is identical to `L.Convolution2D`. But I think we need to not to do so because there is   Actually, `__init__` of `L.Convolution2D` is redundant because of backward compatibility (e.g. the roles of `nobias` `bias`, and `initial_bias` are somewhat overlapped, the role of `wscale` is merge to the initializer fed to `initialW`). We can substitute `Convolution2D` with `ConvolutionND` (or vise versa) if their APIs are same. But I think there is little chance that we want to do so.